src/sinks/influxdb/metrics.rs

```diff
@@ -759,9 +760,9 @@ mod integration_tests {
 
     #[test]
     fn influxdb2_metrics_put_data() {
-        onboarding_v2();
-
         let mut rt = runtime();
+        rt.block_on_std(onboarding_v2());
+
```

Comment:
`fn onboarding_v2` is a test function. Could the `block_on` be moved into it instead of needing to repeat it everywhere it is called?